File: ene-jun-2022/BereniceMaldonado/Practica4.java
package algoritmos;

public class Practica4 {

    public static void main(String[] args) {
        String arr[]={"CANSARTAN HIDROCLOROTIAZIDA ALTER","CARBAMAZEPINA NORMON","MATRIF","ACICLOVIR SANDOZ","MELOXICAM","FENDIVIA","SEBUMSELEN"};
        for(int i=0; i<arr.length-1; i++){
            for(int j=i+1; j<arr.length; j++){
                if(arr[i].length() > arr[j].length()){
                    String temp = arr[i];
                    arr[i] = arr[j];
                    arr[j] = temp;
                }
            }
        }
        for (String arr1 : arr) {
            System.out.println(arr1);
            System.out.println();
        }        
    }
}

File: ene-jun-2023/MiguelAlejandroAnguiano/Practica4.py
arr=[1432,230,1980,1689,400]
print(arr)

for i in range(0,len(arr)):
    for j in range(0,len(arr)-1):
        if arr[j] < arr[j+1]:
            temp=arr[j]
            arr[j]=arr[j+1]
            arr[j+1]=temp
        i+=1
        j+=1
print(arr)